app/javascript/schools/courses/components/curriculum_editor/CurriculumEditor__MarkdownBlockEditor.res

```diff
@@ -10,7 +10,7 @@ let make = (~markdown, ~contentBlock, ~updateContentBlockCB) =>
   <MarkdownEditor
     value=markdown
     profile=Markdown.Permissive
-    maxLength=10000
+    maxLength=500000
```

Comment:
@harigopal I'm unclear on how to best move this rescript value into an environment variable. Is there a way to reference the server side ENV variable here at compile time? Or do I need a separate environment variable for the client side?